openstack/swift-utils/values.yaml

```diff
@@ -1,9 +1,16 @@
+# NOTE: This chart usually consumes the same values.yaml as the "swift" chart.
+
```

Comment:
They don't anymore. We split the values files several years ago, so this is a good opportunity to remove this note.